src/common/database-service/DatabaseMetricsExploreActions.ts

```diff
@@ -41,7 +41,7 @@ export class DatabaseMetricsExploreActions extends DatabaseActions {
       SELECT ${expression} as value, "${column}" as label from "${table}"
       WHERE "${column}" IS NOT NULL ${whereClause}
       GROUP BY "${column}"
-      ORDER BY value desc
+      ORDER BY value desc NULLS LAST
```

Comment:
If we are filtering out nulls then we should do that here.

Reply:
yeah, maybe in hindsight filtering out is the better move

Reply:
Maybe we can show the null labels but make them un-selectable? That way there is no loss of information.